go.mod

```diff
@@ -41,24 +41,27 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.1
 	gotest.tools v2.2.0+incompatible
-	k8s.io/api v0.26.3
-	k8s.io/apimachinery v0.26.3
-	k8s.io/cli-runtime v0.26.3
-	k8s.io/client-go v0.26.3
-	k8s.io/code-generator v0.26.3
-	k8s.io/utils v0.0.0-20230313181309-38a27ef9d749
+	k8s.io/api v0.27.0
+	k8s.io/apimachinery v0.27.0
+	k8s.io/cli-runtime v0.27.0
+	k8s.io/client-go v0.27.0
+	k8s.io/code-generator v0.27.0
+	k8s.io/utils v0.0.0-20230406110748-d93618cff8a2
 	sigs.k8s.io/controller-runtime v0.14.6
 	sigs.k8s.io/controller-tools v0.11.3
-	sigs.k8s.io/kustomize/api v0.12.1
-	sigs.k8s.io/kustomize/kyaml v0.13.9
+	sigs.k8s.io/kustomize/api v0.13.2
+	sigs.k8s.io/kustomize/kyaml v0.14.1
 	sigs.k8s.io/yaml v1.3.0
 )
 
+// TODO: when controller-runtime past v0.14.6 is released, remove this line
+replace sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.13.1-0.20230418125457-f83169592e24
```

Comment:
@laverya the main branch still using k8s 1.26 see https://github.com/kubernetes-sigs/controller-runtime/blob/f83169592e24f80c1704b4c5bf5022620313a271/go.mod#L21-L26

Reply:
yeah, but it implemented the relevant interfaces in https://github.com/kubernetes-sigs/controller-runtime/pull/2223